src/app/components/Figure/Copyright/index.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { string } from 'prop-types';
+import styled from 'styled-components';
+import {
+  FF_NEWS_SANS_REG,
+  C_WHITE,
+  C_EBON,
+  GEL_SPACING,
+} from '../../../lib/constants/styles';
+import { T_MINION } from '../../../lib/constants/typography';
+
+const copyrightSymbolPrefix = '©';
```

Comment:
@pjlee11 you asked a previous question about using the encoded version. Is there any reason to not?

Reply:
We should use the Unicode copyright char, if that works in the footer.